src/frontend/komponenter/Fagsak/Behandlingskort/Behandlingskort.tsx

```diff
@@ -35,49 +36,49 @@ interface IBehandlingskortProps {
 
 const hentResultatfarge = (behandlingResultat: BehandlingResultat) => {
     if (erBehandlingHenlagt(behandlingResultat)) {
-        return AGray300;
+        return ABorderSubtle;
     }
 
     switch (behandlingResultat) {
         case BehandlingResultat.INNVILGET:
         case BehandlingResultat.DELVIS_INNVILGET:
         case BehandlingResultat.FORTSATT_INNVILGET:
-            return AGreen600;
+            return ABorderSuccess;
         case (BehandlingResultat.ENDRET_UTBETALING, BehandlingResultat.ENDRET_UTEN_UTBETALING):
-            return ABlue700;
+            return ABorderAction;
         case BehandlingResultat.AVSLÅTT:
         case (BehandlingResultat.OPPHØRT, BehandlingResultat.FORTSATT_OPPHØRT):
-            return ARed600;
+            return ABorderDanger;
         case BehandlingResultat.IKKE_VURDERT:
-            return AGray100;
+            return ABorderSubtle;
         default:
-            return AGray600;
+            return ABorderDefault;
     }
 };
 
 const hentResultatfargeTekst = (behandlingResultat: BehandlingResultat) => {
     if (erBehandlingHenlagt(behandlingResultat)) {
-        return AGray900;
+        return ATextDefault;
     }
 
     switch (behandlingResultat) {
         case BehandlingResultat.INNVILGET:
         case BehandlingResultat.DELVIS_INNVILGET:
         case BehandlingResultat.FORTSATT_INNVILGET:
-            return AGreen600;
+            return AIconSuccess;
         case (BehandlingResultat.ENDRET_UTBETALING, BehandlingResultat.ENDRET_UTEN_UTBETALING):
-            return ABlue700;
+            return AIconInfo;
         case BehandlingResultat.AVSLÅTT:
         case (BehandlingResultat.OPPHØRT, BehandlingResultat.FORTSATT_OPPHØRT):
-            return ARed600;
+            return ATextDanger;
         default:
-            return AGray900;
+            return ATextDefault;
     }
 };
 
 const Container = styled.div<{ behandlingResultat: BehandlingResultat }>`
-    border: 1px solid ${AGray400};
-    border-left: 0.5rem solid ${AGray400};
+    border: 1px solid ${ABorderSubtle};
```

Comment:
Kunne ha vært interessant å sette 
```
border-left-color: ${({ behandlingResultat }) => hentResultatfarge(behandlingResultat)};
`;
```
Slik at hele boksen får én kantfarge hele veien rundt. Eksempel:

Før:
<img width="344" alt="image" src="https://github.com/navikt/familie-ba-sak-frontend/assets/2379098/8ca1304e-9ff6-4a86-9767-0ea73e8170c1">

Etter:
<img width="290" alt="image" src="https://github.com/navikt/familie-ba-sak-frontend/assets/2379098/d7397e7d-f6cf-4eca-8e8d-df9cf70197fc">

Reply:
Litt usikker på om jeg skjønte hva du mente her, for det kodeforslag er der allerede 🤔 men resultatet ser veldig nice ut da, veldig for å la kanten rundt hele boksen få samme farge 🙌

Reply:
Ah, skrev feil og mente `border-color`! Det er det du ser bilder av

Reply:
5a64d8d2